src/hw10/FileNavigator.java

```diff
@@ -0,0 +1,36 @@
+package hw10;
+
+import java.io.File;
+import java.util.*;
+import java.util.function.BiFunction;
+
+public class FileNavigator extends FileData{
+    static BiFunction<String, String, String> bFunc = (oldValue, newValue)->oldValue+";"+newValue;
+
+    static Map<String, String> listName = new HashMap<>();
```

Comment:
It's better to give more appropriate name to the fields. In this case can be like listName -> map, filesByPath, fileMap, ...